src/components/auth/AuthForm.tsx

```
@@ -57,8 +63,22 @@ interface EmailPasswordFormProps {
 }
 
 function EmailPasswordForm({ isLogin }: EmailPasswordFormProps) {
+  const [fullName, setFullName] = useState("");
```

ESLint

[error] 'setFullName' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)